pkg/common/utils_test.go

```diff
@@ -116,3 +118,18 @@ func TestConvertSITimeout(t *testing.T) {
 		})
 	}
 }
+
+func TestGetIgnoreUnschedulable(t *testing.T) {
```

Comment:
can we cover the case where the tag value is not true or false?

Reply:
Sure